resources/public/api/conf/7.0/foreign_property_accounting_adjustments_submit.yaml

```diff
@@ -72,15 +75,24 @@ post:
           TY-2024-25 - Foreign Property (Non-FHL) with non-negative residential financial cost:
             value:
               $ref: ./examples/submit_foreign_property_accounting_adjustments/def2/submit_foreign_property_accounting_adjustments.json
-          TY-2025-26-Foreign Property Adjustment:
+          TY-2025-26 - Foreign Property Adjustment:
             value:
               $ref: ./examples/submit_foreign_property_accounting_adjustments/def3/submit_foreign_property_accounting_adjustments.json
-          TY-2025-26-Foreign Property Consolidated Adjustment:
+          TY-2025-26 - Foreign Property Consolidated Adjustment:
             value:
               $ref: ./examples/submit_foreign_property_accounting_adjustments/def3/submit_foreign_property_consolidated_accounting_adjustments.json
-          TY-2025-26 onwards - Foreign Property Zero Adjustments:
+          TY-2025-26 - Foreign Property Zero Adjustments:
             value:
               $ref: ./examples/submit_foreign_property_accounting_adjustments/def3/submit_foreign_property_accounting_adjustments_zero.json
+          TY-2026-27 onwards - Foreign Property Adjustment:
```

Comment:
could we hide these based on the feature switch in the same way you have done for the def4 schema pls